### PR TITLE
Kolla image tag bump fixes

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

File: etc/kayobe/kolla/globals.yml [hunk 1]
```diff
@@ -9,7 +9,6 @@ enable_docker_repo: "{% raw %}{{ 'overcloud' not in group_names or ansible_facts
 # kolla_base_distro must be set here to be resolvable on a per-host basis
 # This is necessary for os migrations where mixed clouds might be deployed
 kolla_base_distro: "{% raw %}{{ 'centos' if ansible_facts.distribution == 'Rocky' and ansible_facts.distribution_major_version == '8' else ansible_facts.distribution | lower }}{% endraw %}"
-opensearch_tag: yoga-20230904T121340
 
 # Live-migration is broken from qemu-kvm-6.2.0-20.module_el8.7.0+1218+f626c2ff to qemu-kvm-6.2.0-39.module_el8+669+76cc32af
 # with signature: `Missing section footer for 0000:00:01.3/piix4_pm`. Test nova carefully before bumping.
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml]: Remove this one?

Comment (Alex-Welsh) [reply etc/kayobe/kolla/globals.yml]: I haven't done any additional testing, so thought it would still be required

Comment (markgoddard) [reply etc/kayobe/kolla/globals.yml]: We have it on l21

Comment (Alex-Welsh) [reply etc/kayobe/kolla/globals.yml]: Ah, I see now. Fixed

**Review by markgoddard — APPROVED** [review]

**Pull request merged**